docs/api.md

```diff
@@ -496,6 +496,93 @@ await webcontainerInstance.fs.writeFile(
 );
 ```
 
+### ▸ `watch`
+
+Watch for changes to a given file or directory.
+
+<h4 id="watch-signature">
+  <a id="watch-signature">Signature</a>
+  <a href="#watch-signature" class="header-anchor" aria-hidden="true">#</a>
+</h4>
+
+`watch(path: string, options: Options, Callback): Watcher`
+
+`watch(path: string, Callback): Watcher`
+
+<h4 id="watch-options">
+  <a id="watch-options"><code>Options</code></a>
+  <a href="#watch-options" class="header-anchor" aria-hidden="true">#</a>
+</h4>
+
+```ts
+interface Options {
+  encoding?: BufferEncoding | null;
+  persistent?: boolean;
+  recursive?: boolean;
+}
+```
+
+<br />
+
+#### `encoding?: boolean`
+
+Specifies the character encoding to be used for the filename passed to the listener. Default: `'utf8'`.
+
+#### `persistent?: boolean`
+
+Indicates whether the process should continue to run as long as files are being watched. Default: `true`.
```

Comment:
Which process? Maybe we should provide an example for this flag?

Reply:
I'm not actually sure what effect this flag has, since the code is being run from the main thread anyway

Reply:
Yeah let's remove it.